lib/ProductOpener/Products.pm

```diff
@@ -2414,6 +2419,23 @@ sub index_product($product_ref)
 	return;
 }
 
+sub index_search_service($product_ref) {
+	if ($ProductOpener::Config2::redis_url eq "")
+	{
+		$log->warn("Redis URL not provided for search indexing", { error => $@ }) if $log->is_warn();
```

Comment:
We should only warn about this once, eg at startup (at least not on every request).
(easier to do if we keep the connection, as proposed in another comment)